src/main/java/page_objects/PageObject.java

```diff
@@ -0,0 +1,16 @@
+package page_objects;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class PageObject {
+
+    protected WebDriver driver;
+
+    public PageObject(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+}
```

Comment:
please remove empty lines at the end of the class

Reply:
done